tests/unit/test_bronze_tier_compliance.py

```
@@ -218,6 +218,71 @@ async def test_appropriate_polling_interval(self, hass, api_ok):
+        try:
+            from PIL import Image
+        except ImportError:
+            pytest.skip("Pillow not installed, cannot validate image format/dimensions")
+
+        # Verify icon.png
+        with Image.open(icon_path) as img:
+            assert img.format == "PNG", f"icon.png must be PNG format, got {img.format}"
+            assert img.size == (
+                256,
+                256,
+            ), f"icon.png must be 256×256 pixels, got {img.size}"
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)